Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -69,12 +82,33 @@ public void StartTurn()
 
     protected void TakeDamage(int damage)
     {
-        CurrentHealthPoints -= damage;
+        if (CurrentShieldPoints > 0)
+        {
+            CurrentShieldPoints -= damage;
+            if (CurrentShieldPoints <= 0) {
+                Shield.SetActive(false);
+                int leftoverDmg = -1 * CurrentShieldPoints;
+                CurrentHealthPoints -= leftoverDmg;
+            }
+        }
+
+        else
+            CurrentHealthPoints -= damage;
         
         if (CurrentHealthPoints <= 0)
             IsAlive = false;
     }
 
+    public void ApplyShield(int shieldHealth)
+    {
+        Shield.transform.position = GameObject.FindGameObjectWithTag("Player").transform.position;
+        Shield.transform.position -= new Vector3(0, 5, 0);
+        Shield.transform.Translate(new Vector3(0, 5, 0));
```

Comment:
Uhhh this line just did what the above line did? Looks like a bug

Reply:
Yeah it does...whoops. Good catch! Fixed it accordingly